modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install

```diff
@@ -381,23 +381,22 @@ function social_group_flexible_group_update_8912() {
  * Allow CM+ translate "Flexible Group" group type.
  */
 function social_group_flexible_update_10301() {
-  foreach (['sitemanager', 'contentmanager'] as $role_id) {
-    user_role_grant_permissions($role_id, ['translate flexible_group group']);
-  }
+  // Removed hook due to that it doesn't apply since wrong current function
+  // name. It should be social_group_flexible_group_update_10301. New hook
+  // update was added.
+  // @see social_group_flexible_group_update_11001()
+  // @see https://www.drupal.org/project/social/issues/3273182
 }
 
 /**
  * Update view display for Flexible Group.
  */
 function social_group_flexible_update_10302() {
-  /** @var \Drupal\update_helper\Updater $updateHelper */
-  $updateHelper = \Drupal::service('update_helper.updater');
-
-  // Execute configuration update definitions with logging of success.
-  $updateHelper->executeUpdate('social_group', 'social_group_flexible_group_update_10302');
-
-  // Output logged messages to related channel of update execution.
-  return $updateHelper->logger()->output();
+  // Removed hook due to that it doesn't apply since wrong current function
```

Comment:
Same as above, just change the update hook name.

Reply:
Updated